Solution1/BusinessLayer/Services/WeatherService.cs

```diff
@@ -127,5 +135,34 @@ public async Task<ForecastWeatherDTO> GetForecastByCityNameAsync(string cityName
                             .ToDictionary(k => k.Key, v => v.Select(response => response));
             return result;
         }
+
+        public async Task BackgroundSaveWeatherAsync(IEnumerable<string> cities)
```

Comment:
this method does nothing in the background: so it shouldn't be in name

Reply:
+